rcp/org.testeditor.rcp4.views.tcltestrun.tests/src/org/testeditor/rcp4/views/tcltestrun/TestResultFileWriterTest.xtend

```diff
@@ -0,0 +1,97 @@
+package org.testeditor.rcp4.views.tcltestrun
+
+import com.google.common.io.Files
+import java.io.File
+import java.nio.charset.StandardCharsets
+import javax.xml.parsers.DocumentBuilderFactory
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.InjectMocks
+import org.mockito.runners.MockitoJUnitRunner
+
+import static org.junit.Assert.*
+
+@RunWith(MockitoJUnitRunner)
+class TestResultFileWriterTest {
+
+	@InjectMocks
+	TestResultFileWriter testResultWriter
+
+	@Test
+	def void testErrorFile() {
+		// given
+		val errorFile = File.createTempFile("mFile", "tmp")
+		// when
+		testResultWriter.writeErrorFile("MyPrj.package.MyTest", errorFile)
+		// then		
+		val lines = Files.readLines(errorFile, StandardCharsets.UTF_8)
```

Comment:
Why not `Files.toString(...)` and then check against the full contents? That would require us to extract the contents of the error file to a static constant which is a good idea anyway.